xblockutils/public/studio_edit.js

```diff
@@ -65,6 +66,11 @@ function StudioEditableXBlockMixin(runtime, element) {
                 }
             });
         }
+
+        if (type == 'datepicker' && datepickerAvailable) {
+            $field.datepicker('destroy');
+            $field.datepicker({dateFormat: "m/d/yy"});
```

Comment:
After looking at how the datepicker is already used in edx-platform, it seems like nothing is done on that front. And the `month/day/year` format is hardcoded eveywhere.

I just want to point out, maybe for later, that `m/d/y` can be extremely confusing for people that use `d/m/y` format (and the other way around, probably :wink:).